### DISCUSSION TITLE
Hello From Suwanee, GA

### DISCUSSION THREAD

**Discussion by ghost** [community/community#56450] [GitHub Education]
### Hi everyone,

Hi everyone. I am a new GitHub subscriber learning Python. Looking forward to engaging conversation and posts.

**Comment by samus-aran** [comment]
👋 Welcome to the GitHub Community on Discussions, @lescorbett66  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
